tutorial/02_segmentation_and_measuring.md

```diff
@@ -1,494 +1,598 @@
 
-### SLIC
+points.data = np.array(
+      [[ 30.        ,  14.2598685 ,  27.7741219 ],
+       [ 30.        ,  30.10416663,  81.36513029],
+       [ 30.        ,  13.32785096, 144.27631406],
+       [ 30.        ,  46.8804823 , 191.80920846],
+       [ 30.        ,  43.15241215, 211.84758551],
+       [ 30.        ,  94.87938547, 160.12061219],
+       [ 30.        ,  72.97697335, 112.58771779],
+       [ 30.        , 138.21820096, 189.01315585],
+       [ 30.        , 144.74232372, 242.60416424],
+       [ 30.        ,  98.14144685, 251.92433962],
+       [ 30.        , 153.59649032, 112.58771779],
+       [ 30.        , 134.49013081,  40.35635865],
+       [ 30.        , 182.95504275,  48.74451649],
+       [ 30.        , 216.04166532,  80.89912152],
+       [ 30.        , 235.14802483, 130.296051  ],
+       [ 30.        , 196.00328827, 169.44078757],
+       [ 30.        , 245.86622651, 202.06140137],
+       [ 30.        , 213.71162148, 250.52631331],
+       [ 28.        ,  87.42324517,  52.00657787]],
+      dtype=float,
+)
+```
 
-There are many analogies to machine learning in unsupervised segmentation.  Our first example directly uses a common machine learning algorithm under the hood - K-Means.
+Once you have marked all the points, you can grab the data back, and make a markers image for `skimage.segmentation.watershed`:
 
 ```{code-cell} ipython3
-# SLIC works in color, so we will use the original astronaut
-astronaut_slic = seg.slic(astronaut)
+from skimage import segmentation
+
+marker_locations = points.data
+
+markers = np.zeros(nuclei.shape, dtype=np.uint32)
+marker_indices = tuple(np.round(marker_locations).astype(int).T)
+markers[marker_indices] = np.arange(len(marker_locations)) + 1
+markers_big = morphology.dilation(markers, morphology.ball(5))
+
+segmented = segmentation.watershed(
+    edges,
+    markers_big,
+    mask=remove_objects,
+)
+
+viewer.add_labels(
+    segmented,
+    scale=spacing,
+)
+
+viewer.layers['labels'].visible = False
 ```
 
 ```{code-cell} ipython3
-# label2rgb replaces each discrete label with the average interior color
-image_show(color.label2rgb(astronaut_slic, astronaut, kind='avg'));
+:tags: [remove-input]
+
+nbscreenshot(viewer)
 ```
 
-We've reduced this image from 512*512 = 262,000 pixels down to 100 regions!
+After watershed, we have better disambiguation between internal cells!
 
-And most of these regions make some logical sense.
++++
+
+## Making measurements
 
 +++
 
-### Chan-Vese
+Once we have defined our objects, we can make measurements on them using `skimage.measure.regionprops` and the new `skimage.measure.regionprops_table`. These measurements include features such as area or volume, bounding boxes, and intensity statistics.
```

Comment:
```suggestion
Once we have defined our objects, we can make measurements on them using `skimage.measure.regionprops` and `skimage.measure.regionprops_table`. These measurements include features such as area or volume, bounding boxes, and intensity statistics.
```
Not sure if you still want to call it new as it was added in 0.16?

Reply:
hahaha very good 😂